ivy_tests/test_ivy/test_stateful/test_norms.py

```diff
@@ -1,56 +1,64 @@
 """Collection of tests for normalization layers."""
 
 # global
-import pytest
 import numpy as np
+from hypothesis import given
+from hypothesis import strategies as st
 
 # local
 import ivy
-from ivy.container import Container
 import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
 
 
 # layer norm
-@pytest.mark.parametrize(
-    "x_n_ns_n_target",
-    [
-        (
-            [[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]],
-            [3],
-            [[-1.2247356, 0.0, 1.2247356], [-1.2247356, 0.0, 1.2247356]],
-        ),
-    ],
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    new_std=st.floats(min_value=0.0, max_value=1.0),
+    init_with_v=st.booleans(),
+    method_with_v=st.booleans(),
+    num_positional_args_init=helpers.num_positional_args(fn_name="LayerNorm.__init__"),
```

Comment:
I think we don't need to use this helper as we have pre-defined positional-only and keyword-only arguments, please pass the number of positional-only args to the test_method as a constant directly.